### DISCUSSION TITLE
编写这样一个自动化脚本需要学习哪些东西

### DISCUSSION THREAD

**Discussion by mengzhi73** [MAA1999/M9A#79] [Q&A] [answered]
有没有大佬给个学习路线呀，孩现在大一只学过c++和python基础，不知道该学啥了

**Comment by xc37a** [comment]
> 有没有大佬给个学习路线呀，孩现在大一只学过c++和python基础，不知道该学啥了

同问

**Comment by MistEO** [comment, marked as answer]
可以参考 https://github.com/MaaXYZ/MaaFramework/blob/main/docs/zh_cn/1.1-%E5%BF%AB%E9%80%9F%E5%BC%80%E5%A7%8B.md
